features/addons/src/main/feature/feature.xml

```diff
@@ -3,10 +3,13 @@
 
 	<!-- these are 2.x add-ons in separate repositories (which do not have their own Karaf feature defined), so we include them here -->
 
-    <feature name="openhab-binding-zigbee" description="ZigBee Binding" version="${project.version}">
+    <feature name="openhab-binding-zigbee-ember" description="ZigBee Binding for Ember NCP" version="${project.version}">
```

Comment:
Why did you rename this feature? [My suggestion](https://github.com/openhab/org.openhab.binding.zigbee/issues/47#issuecomment-366700799) was to leave things as they are in the face of the user and just add all the different bundles in this feature.

Reply:
Ah - ok, I misunderstood. no problem, I can add all 4 - no problem.